qfieldsync/ui/package_dialog.ui

```diff
@@ -17,7 +17,7 @@
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
-      <number>0</number>
+      <number>1</number>
      </property>
```

Comment:
Be careful with these, this should be reverted. 

This happens when you open QtDesigner and you click the different tabs/stacked widgets and when you save, it keeps the last opened tab/page. You should keep the original page and never change this unless you have a good reason to do so.

Reply:
Noted!